rpmspec/package_venv.spec

```diff
@@ -6,18 +6,32 @@
 %define venv_install_dir opt/stackstorm/%{venv_name}
 %define venv_dir %{buildroot}/%{venv_install_dir}
 %define venv_bin %{venv_dir}/bin
+
+%if 0%{?rhel} == 8
+%define venv_python %{venv_bin}/python3
+%define install_crypto %{venv_python} %{venv_bin}/pip install cryptography==2.8 --no-binary cryptography
+%define install_venvctrl python3 -m pip install venvctrl
+%else
 %define venv_python %{venv_bin}/python
+%define install_crypto %{nil}
+%define install_venvctrl %{nil}
+%endif
+
 %define venv_pip %{venv_python} %{venv_bin}/pip install --find-links=%{wheel_dir} --no-index
 
-# Change the virtualenv path to the target installation direcotry.
+# Change the virtualenv path to the target installation directory.
 #   - Install dependencies
 #   - Install package itself
+
+# EL8 requires crypto built locally and venvctrl available outside of venv
 %define pip_install_venv \
-  %if 0%{?use_st2python} \
```

Comment:
Can we stick with original indentation? It's easier to read it.

Reply:
sure.